packages/test-wasm/src/browser_shim.ts

```diff
@@ -1,3 +1,5 @@
+import { createDatabase } from '@0x/mesh-browser-lite/lib/database';
 import { createSchemaValidator } from '@0x/mesh-browser-lite/lib/schema_validator';
 
 (window as any).createSchemaValidator = createSchemaValidator;
+(window as any).__mesh_dexie_newDatabase__ = createDatabase;
```

Comment:
Since we are assigning to `window`, I added a prefix and suffix to make it less likely to collide with variable names from other packages. I'll do the same for `createSchemaValidator`.